widget/embedded/src/pages/Home.tsx

```diff
@@ -189,6 +189,9 @@ export function Home() {
                       USD_VALUE_MIN_DECIMALS,
                       USD_VALUE_MAX_DECIMALS
                     ),
+                realUsdValue: priceImpactInputCanNotBeComputed
+                  ? undefined
+                  : formatTooltipNumbers(outputUsdValue),
```

Comment:
I believe the condition of this code is not correct.
 you can check this routes on Solana Blockchain :
 sol -> wsol
 wsol->sol
 
![image](https://github.com/rango-exchange/rango-client/assets/120931880/44a50e39-f2ff-4017-9b0a-a662dabb7cf0)

Reply:
fixed.